library/src/main/java/com/alamkanak/weekview/WeekView.java

```diff
@@ -1085,6 +1096,9 @@ private void drawEventTitle(WeekViewEvent event, RectF rect, Canvas canvas, floa
         int availableWidth = (int) (rect.right - originalLeft - mEventPadding * 2);
 
         // Get text dimensions.
+        if(textColorPicker != null) {
```

Comment:
Yes, that's right. I was thinking about how to use it with the setTextColor methods, but that isn't possible. So this is right.